Including new IMAGE_TRUST_CHECKS makefile flag (#5786)
wazuh/wazuh-documentation: source/development/makefile.rst

| **DISABLE_CISCAT**        | Disable the compilation of the CIS-CAT module.                                                                                                       |
|                           +------------------+-----------------------------------------------------------------------------------------------------------------------------------+
|                           | Default value    | n/a                                                                                                                               |
|                           +------------------+-----------------------------------------------------------------------------------------------------------------------------------+
|                           | Allowed values   | 1, yes, YES, y, Y                                                                                                                 |
+---------------------------+------------------+-----------------------------------------------------------------------------------------------------------------------------------+

> | **IMAGE_TRUST_CHECKS**    | Configures the action to take when a library is not trusted in Windows. It can be disabled (0), only generate warning (1) or shutdown the agent (2). |

Reviewer: ```suggestion
| **IMAGE_TRUST_CHECKS**    | Configures the action to take when a library is not trusted in Windows. Actions allowed are: Disabled (0), Only generate warning (1), and Shutdown the agent (2). |
```
